### PR TITLE
Adds a configurable message to be sent to mobs on polymorph

### PR DESCRIPTION
**Pull request by Cyberboss**
Closes #28472

### CODE REVIEW

**Comment by lzimann** [comment]
The span should probably be in the code instead.

**Comment by Cyberboss** [comment]
I'd rather not hide the fact that what we're sending is html and can be configured as such

File: config/polymoprh_rules.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+<span class='danger'>Note that you are allowed to act as an antagonist while transformed into a hostile mob, unless you volunteered for or seeked out transformation.</span>
```
Comment (deathride58) [inline config/polymoprh_rules.txt]: Filename is misspelled. Code looks for polymorph_rules.txt, not polymoprh_rules.txt

**Comment by Sligneris** [comment]
> poly_policy

Makes me think about the parrot instead.

File: code/modules/projectiles/projectile/magic.dm [hunk 2]
```diff
@@ -277,6 +277,10 @@
 
 	to_chat(new_mob, "<span class='warning'>Your form morphs into that of a [randomize].</span>")
 
+	var/static/poly_policy = trim(file2text("config/polymorph_rules.txt"))
+	if(poly_policy)
+		to_chat(new_mob, poly_policy)
```
Comment (Sligneris) [inline code/modules/projectiles/projectile/magic.dm]: \>`poly_policy`

Makes me think about the parrot instead.

File: code/modules/projectiles/projectile/magic.dm [hunk 3]
```diff
@@ -277,6 +277,10 @@
...
+
```
Comment (Sligneris) [inline code/modules/projectiles/projectile/magic.dm]: \>`poly_policy`

Makes me think about the parrot instead.

**Comment by PKPenguin321** [comment]
It just occurred to me, should the message be "seeked out" or "sought out"

**Comment by optimumtact** [comment]
sought out sounds natural to me

**Comment by RemieRichards** [comment]
sought out.

**Comment by AnturK** [comment]
I'd just make is some generic server_policy file. So in the future something that seems affected by rules can be put there without filling the config directory with one line text files.

**Comment by PKPenguin321** [comment]
Just noticed that you made a policy system thing for this. You should move the drone BIG RED TEXT describing its policy to this system

**Comment by Cyberboss** [comment]
Very observant

**Pull request merged**